test (internal): Failure when run in MST timezone

### Relevant telegraf.conf

```toml
n/a
```


### Logs from Telegraf

```text
--- FAIL: TestTimestampAbbrevWarning (0.00s)
    internal_test.go:766: 
        	Error Trace:	/home/powersj/telegraf/internal/internal_test.go:766
        	Error:      	"" does not contain "Your config is using abbreviated timezones and parsing was changed in v1.27.0"
        	Test:       	TestTimestampAbbrevWarning
FAIL
FAIL	github.com/influxdata/telegraf/internal	0.003s
FAIL
```


### System info

master

### Docker

_No response_

### Steps to reproduce

Run tests when set to MST timezone.

### Expected behavior

test pass

### Actual behavior

test fail

### Additional info

_No response_

--- a/internal/internal_test.go
+++ b/internal/internal_test.go
@@ -759,9 +759,15 @@ func TestTimestampAbbrevWarning(t *testing.T) {
 	log.SetOutput(&buf)
 	defer log.SetOutput(backup)
 
+	// Try multiple timestamps with abbreviated timezones in case a user
+	// is actually in one of the timezones.
 	ts, err := ParseTimestamp("RFC1123", "Mon, 02 Jan 2006 15:04:05 MST", nil)
 	require.NoError(t, err)
 	require.EqualValues(t, 1136239445, ts.Unix())
+
+	ts2, err := ParseTimestamp("RFC1123", "Mon, 02 Jan 2006 15:04:05 EST", nil)
+	require.NoError(t, err)
+	require.EqualValues(t, 1136232245, ts2.Unix())
 
 	require.Contains(t, buf.String(), "Your config is using abbreviated timezones and parsing was changed in v1.27.0")
 }

